### DISCUSSION TITLE
Cannot get custom domain to not throw error even when DNS resolves

### DISCUSSION THREAD

**Discussion by tknx** [community/community#52282] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am trying to stand up a standard Jekyll install on Github Pages. It all works fine except the Google-hosted custom domain name throws an error.

The website itself seems to resolve fine when I go to www.nagpal.com.

Here is an image of the error:
![image](https://user-images.githubusercontent.com/29260341/230663442-6c9c6651-57f6-49f5-a7f6-85b8a9c31f2c.png)

And here is an image of my Google DNS settings;
![image](https://user-images.githubusercontent.com/29260341/230663497-16f4d915-bdce-4a20-ae39-4726d870e4ce.png)
![image](https://user-images.githubusercontent.com/29260341/230663509-77500aa8-86e9-4bae-ba9b-1c8cdcc59136.png)

**Comment by airtower-luna** [comment]
How long have the DNS records been configured like this? It can take quite a while for changes to move through the caches.

**Comment by tknx** [comment]
Yep - I must have been impatient and not waited long enough. It just worked...
